pexpect/spawnbase.py

```diff
@@ -140,6 +143,15 @@ def _coerce_send_string(self, s):
             return s.encode('utf-8')
         return s
 
+    def _get_buffer(self):
+        return self._buffer.getvalue()
+
+    def _set_buffer(self, value):
+        self._buffer = self.buffer_type()
+        self._buffer.write(value)
+
+    buffer = property(_get_buffer, _set_buffer)
```

Comment:
Could we have a comment to remind anyone working on the code that this interface as a string is required for backwards compatibility?